app/views/schools/add_participants/_ect_or_mentor_answers.html.erb

```diff
@@ -0,0 +1,94 @@
+<%= govuk_summary_list do |summary_list| %>
+  <% summary_list.row do |row| %>
+    <% row.key { "Name" } %>
+    <% row.value { form.full_name } %>
+    <% unless form.found_participant_in_dqt? %>
+      <% row.action(text: "Change",
+                    visually_hidden_text: "name",
+                    href: form.change_path_for(step: :name)) %>
+    <% else %>
+      <% row.action() %>
+    <% end %>
+  <% end %>
+
+  <% if form.trn %>
+   <% summary_list.row do |row| %>
+       <% row.key { "TRN" } %>
+       <% row.value { form.trn } %>
+       <% unless form.found_participant_in_dqt? %>
+         <% row.action(text: "Change",
+                       visually_hidden_text: "TRN",
+                       href: form.change_path_for(step: :trn)) %>
+       <% else %>
+         <% row.action() %>
+       <% end %>
+     <% end %>
+   <% end %>
+
+  <% if form.date_of_birth %>
+    <% summary_list.row do |row| %>
+      <% row.key { "Date of birth" } %>
+      <% row.value { form.date_of_birth.to_date.to_s(:govuk) } %>
+      <% unless form.found_participant_in_dqt? %>
+        <% row.action(text: "Change",
+                      visually_hidden_text: "date of birth",
+                      href: form.change_path_for(step: :date_of_birth)) %>
+      <% else %>
+        <% row.action() %>
```

Comment:
More recent versions of the summary list don't need a blank action, should be safe to drop (and if not we're best upgrading `govuk-frontend`)